### ISSUE TITLE
Did typewriter stop working in VS 16.7.0?

### ISSUE THREAD

**Issue by gregveres** [frhagn/Typewriter#334]
I just upgraded to 16.7.0 (boy do I regret that). And one of the issues I am having is that typewriter doesn't seem to be outputting changes anymore. I see the status bar say Rendering files, but the resulting file doesn't change.

**Comment by gregveres** [comment]
It definitely looks like it has stopped working in VS 16.7.0. 

I uninstalled typewriter and reinstalled it. It seems to run, but it doesn't write any files out. 

It could be just that my VS is screwed. I am having tons of problems with VS 16.7.0 and wish I hadn't upgraded. If anybody else is using VS 16.7.0 or later, please let me know if it is working for you.
thanks.

**Comment by MGundersen** [comment]
Hello @gregveres 

I have just formatted my computer today, and installed the newest version of VS (16.7.0), and typewriter, and I was able to render templates just fine.
I hope you will find a fix to your problem.

Best Regards,
Mathias

**Comment by gregveres** [comment]
@MGundersen Thanks for the feedback Mathias. When I upgraded to 16.7.0 the upgrade failed. I recovered from it, but 16.7.0 was completely busted due to that botched upgrade. 
Last night I uninstalled VS and reinstalled VS, expecting to go back to 16.5, but the installer installed the latest version so I found myself with a fresh 16.7.0. That worked as expected so it must have just been the botched upgrade that caused the issue. 

I will close this since it has nothing to do with typewriter.
